consumer/dragonboard820c/hardware-docs/README.md

```diff
@@ -8,4 +8,4 @@ Explore what makes your DragonBoard 820c unique, technical specifications, schem
 
 ## User Guides
 
-- Hardware User Manual ([View]() / [Download]())
+- Hardware User Manual ([View](https://github.com/96boards/documentation/blob/master/consumer/dragonboard820c/hardware-docs/files/db820c-hw-user-manual.pdf) / [Download](https://github.com/96boards/documentation/raw/master/consumer/dragonboard820c/hardware-docs/files/db820c-hw-user-manual.pdf))
```

Comment:
Please don't reference GitHub links. These should be relative to the 96Boards site.

Reply:
Can a relative link be applied to static files?

AFAIK, the only way to forward users to a static file hosted on the Github is to forward them to the Github location. We have yet to implement a way to render said docs on the website with a relative link.

All of our other boards are using this same method.

@kylekirkby input?

Reply:
If you change the link to be:

/consumer/dragonboard820c/hardware-docs/files/db820c-hw-user-manual.pdf

that will work. Since they are PDFs, no rendering is required? I'm not sure about the view vs download aspect of things. For example, if I click on a PDF link, one of two things is likely to happen - my browser will ask me if I want to save it or it will download it and open it automatically with a PDF viewer. If the latter happens, all viewers I've used have the ability to save that file to disc for keeping.

Reply:
@pcolmer okay, the change as been made for the "View" link. Guessing "Download" link should still be the same?

Reply:
Well, like I said, I think you could get away with a single link and, perhaps, some advisory text that says something like "To download the manual, right click and choose 'Save As'". I'm not a *great* fan of that because the browser wording can vary.

Even if you keep the "download" link, what's to stop a browser from automatically opening it in a PDF viewer engine anyway?